src/TodoList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+
+function TodoList() {
+
+	const todoList = [];
+	todoList.push( {id: 1, title: "Complete assignment"} );
+	todoList.push( {id: 2, title: "Watch video lectures"} );
+	todoList.push( {id: 3, title: "Go to mentor sessions"} );
+	todoList.push( {id: 4, title: "Do mindset exercises"} );
+	todoList.push( {id: 5, title: "Go to sleep"} );
+
+	return (
+		<div>
+			<ul>
+				{/* todoList.map( function (e) {
```

Comment:
Go ahead and take out the commented code.  It will exist in source control if needed.